### DISCUSSION TITLE
Set time to disable email

### DISCUSSION THREAD

**Discussion by lion-share** [simple-login/app#673] [Feature Requests]
Please implement a way to disable/delete an email address after X amount of days/months/etc.

**Comment by swapnil2905** [comment]
@nguyenkims I had requested for a similar feature, We really need this Feature as we are generating too many aliases which stay unused and this can replace the temp email solution as well, kindly implement it ASAP. 
Thanks.

**Reply by nguyenkims** [reply]
Thanks, we'll make sure to discuss about this feature when planning new development.

**Reply by swapnil2905** [reply]
Thank you for the Reply, looking forward to seeing this feature implemented this year!

**Reply by ryebread321** [reply]
Any update? It's been over a year.

**Comment by JoeBoll** [comment]
Ohh I hadn't thought of this! I'd totally use it though.

It would be amazing for public events like trade shows if it could be applied by an auto create alias rule.
